client/pages/ClubList.tsx

```diff
@@ -0,0 +1,19 @@
+import React from "react";
+import { useContext } from "react";
+import ClubCard from "../components";
+import { AppContext } from "./_app";
+
+export default function ClubList() {
+  const appData = useContext(AppContext);
+  const clubListData = appData.clubs;
+
+  return (
+    <div>
+      <h1>Club List</h1>
+      {Array.from(clubListData.entries()).map(([clubId, clubInfo]) => {
```

Comment:
Was looking into this, apparently Array.from is not supported by IE based on the newest ECMAScript standard (as detailed here): https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/from

However, Justin was able to load the club list in IE regardless somehow ??? Probably isn't a concern, but club list may not work on older versions of IE. Might be worth looking into.

Reply:
Actually, we shouldn't be `Array.from`ing anything here. JS maps iterate in insertion order (TIL), but we don't want to use that ordering IRL. ClubListData is pretty much a cache. To control the order in which elements appear, we will need another array-like data structure of club IDs.

Reply:
😮 that makes sense, I guess it's just a set.

The 2 options I can think of atm are:
1. Store the clubs in an array. The con here would be linear time when searching for the club data w/ an id to display on an individual club details page.
2. Convert the map to an array similar to how we're doing here, then sort by some value (maybe alphabetical or id?). Con here would be having to sort the clubs list and finding an alt. to Array.from. 

I think option 1 is the better option imo, but lmk if there are other solutions!

Reply:
We will deprioritize webpack et all for another PR.

Reply:
We will stick with using a map and ensure that the backend sends a sorted list of clubs (alphabetically).